packages/components/src/summary/number.js

```diff
@@ -1,13 +1,16 @@
 /**
  * External dependencies
  */
-import { __, sprintf } from '@wordpress/i18n';
-import { Button } from '@wordpress/components';
+import {
+	Button,
+	Tooltip,
+	__experimentalText as Text,
```

Comment:
TIL! We might be using this tactic for the `TaskList` and `QuickLinks` as well (really the `Card` component I guess)

Does this change to `Text` after a certain version of `@wordpress/components`?

Reply:
>Does this change to Text after a certain version of @wordpress/components?

Thats the idea. I like this component because design now includes the `variant` prop: "title:small", which makes creating text really nice.